### DISCUSSION TITLE
Is it necessary for inline subscriptions to support shared subscriptions?

### DISCUSSION THREAD

**Discussion by werbenhu** [mochi-mqtt/server#387] [General]
Currently, inline subscriptions do not support shared subscriptions. #385 mentioned this question. I personally don't have this kind of use case. @thedevop @mochi-co @dgduncan what do you think? BTW, I personally lean towards inline subscriptions not needing to support shared subscriptions.

**Comment by dgduncan** [comment]
@werbenhu Did they explain what their use case was? I have only used shared-subscriptions in order to load balance a subscription across VMs because the throughput on that topic was too high to keep up with along with the overhead of the work that was being done which I do not see would be the case for the inline client.

**Reply by werbenhu** [reply]
The server subscribes to a certain topic to consume data, while there's also a peripheral system that needs to subscribe to the same topic for consumption. At this point, it's necessary to share the subscription to this topic with the server.

Currently, this requirement isn't very strong; they've addressed it by subscribing to different topics.
